wgsl/index.bs

```diff
@@ -1287,6 +1287,20 @@ An attribute [=shader-creation error|must not=] be specified more than once per
     Specifies a part of the user-defined IO of an entry point.
     See [[#input-output-locations]].
 
+  <tr><td><dfn noexport dfn-for="attribute">`must_use`</dfn>
+    <td>*None*
+    <td>Must only be applied to the declaration of a [=function/function=] with a [=return type=].
+
+        Specifies that a [=function call|call=] to this function [=shader-creation error|must=] be used as an [=expression=].
+        That is, a call to this function [=shader-creation error|must=] not be the entirety of a [[#function-call-statement|function call statement]].
+
+        Note: Many functions return a value and do not have side effects.
+        It is often a programming error to call such a function as the only thing in a function call statement.
+        Such functions are declared as `@must_use`.
```

Comment:
This sentence reads strangely to me. It seems to be making an assertion, but user-declared functions are not required to have this attribute. Suggest either clarifying this is about built-in functions or changing "are" to "should be".

Reply:
How about this:
```
@@ -1296,7 +1296,8 @@ An attribute [=shader-creation error|must not=] be specified more than once per
 
         Note: Many functions return a value and do not have side effects.
         It is often a programming error to call such a function as the only thing in a function call statement.
-        Such functions are declared as `@must_use`.
+        Built-in functions with these properties are declared as `@must_use`.
+        User-defined functions can also have the `@must_use` attribute.
 
         Note: To deliberately work around the `@must_use` rule, use a [=phony assignment=]
         or [=value declaration|declare a value=] using the function call as the initializer.
```

Reply:
SGTM